packages/opentelemetry-id-generator-aws-xray/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@opentelemetry/id-generator-aws-xray",
+  "version": "0.15.0",
+  "description": "AWS X-Ray ID generator for OpenTelemetry",
+  "main": "build/src/index.js",
+  "browser": {
+    "./src/platform/index.ts": "./src/platform/browser/index.ts",
+    "./build/src/platform/index.js": "./build/src/platform/browser/index.js"
+  },
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "precompile": "tsc --version",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.test.ts",
+    "test:browser": "nyc karma start --single-run",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "version": "node ../../scripts/version-update.js",
+    "watch": "tsc --build --watch",
+    "prepare": "npm run compile"
+  },
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "browser",
+    "tracing",
+    "AWS",
+    "X-Ray"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=8.5.0"
+  },
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "devDependencies": {
+    "@types/mocha": "8.2.0",
+    "@types/node": "14.14.12",
+    "@types/sinon": "9.0.4",
+    "@types/webpack-env": "1.16.0",
+    "codecov": "3.8.1",
+    "gts": "3.1.0",
+    "istanbul-instrumenter-loader": "3.0.1",
+    "karma": "5.2.3",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-coverage-istanbul-reporter": "3.0.3",
+    "karma-mocha": "2.0.1",
+    "karma-spec-reporter": "0.0.32",
+    "karma-webpack": "5.0.0",
```

Comment:
Is this working for you as karma 5 doesn't work for me (osx), had to downgrade back to 4